configs/find_threshold_config.yaml

```diff
@@ -0,0 +1,21 @@
+model_type: "IncrFeatStridedConvFCUpsampReflectPadAE"
+model: "/home/fdumais/Desktop/Data/finta_rbx_atlas/models/best_model_contrastive_tractoinferno_hcp.pt"
```

Comment:
Avoid local paths. Use placeholders, relative paths or think of a way to make this work regardless of the file tree. Same applies to all instances below.

Reply:
I modified the scripts to pass to variable as parameters

Reply:
Done